include/aspect/postprocess/binary_data.h

```diff
@@ -5,36 +5,60 @@
 #include <aspect/postprocess/interface.h>
 #include <aspect/simulator_access.h>
 
-namespace aspect {
-    namespace Postprocess {
+#include <boost/serialization/serialization.hpp>
 
-        template<int dim>
-        class BinaryData : public Interface<dim>, public ::aspect::SimulatorAccess<dim> {
-        public:
-            BinaryData();
+namespace aspect
+{
+  namespace Postprocess
+  {
 
-            ~BinaryData();
+    struct Field
```

Comment:
The set of fields that are to be serialized and written out to a user specified file.